lifecycle-operator/webhooks/pod_mutator/handlers/objectmeta.go

```diff
@@ -0,0 +1,121 @@
+package handlers
+
+import (
+	"fmt"
+	"hash/fnv"
+	"strings"
+
+	apicommon "github.com/keptn/lifecycle-toolkit/lifecycle-operator/apis/lifecycle/v1alpha3/common"
+	operatorcommon "github.com/keptn/lifecycle-toolkit/lifecycle-operator/common"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func GetLabelOrAnnotation(resource *metav1.ObjectMeta, primaryAnnotation string, secondaryAnnotation string) (string, bool) {
```

Comment:
if this function and `GetOwnerReference` are currently not used outside of this package, i suggest to make them also unexported like the other ones in this file

Reply:
ok thanks for clarifying, then that's perfectly fine :)